diff --git a/node_modules/@react-pdf/textkit/lib/textkit.js b/node_modules/@react-pdf/textkit/lib/textkit.js
index 0b8b6fa..58b06b8 100644
--- a/node_modules/@react-pdf/textkit/lib/textkit.js
+++ b/node_modules/@react-pdf/textkit/lib/textkit.js
@@ -1030,6 +1030,8 @@ const reorderLine = (line) => {
         for (let i = 0; i < selectedIndices.length; i += 1) {
             const index = selectedIndices[i];
             const glyph = getItemAtIndex(line.runs, 'glyphs', index);
+            if (!glyph)
+                continue;
             if (addedGlyphs.has(glyph.id))
                 continue;
             updatedGlyphs.push(glyph);
